app/src/main/AndroidManifest.xml

```diff
@@ -17,8 +17,7 @@
         <activity
             android:name=".Activities.MainActivity"
             android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar"
-            android:configChanges="orientation|screenSize">
```

Comment:
Why has this been removed?

Reply:
It's still a fresh commit by @flocke in the repo and I haven't updated my fork with it just yet!